reflex/.templates/web/utils/state.js

```diff
@@ -506,7 +518,18 @@ export const hydrateClientStorage = (client_storage) => {
       }
     }
   }
-  if (client_storage.cookies || client_storage.local_storage) {
+  if (client_storage.sessionStorage && typeof window != "undefined") {
```

Comment:
```suggestion
  if (client_storage.session_storage && typeof window != "undefined") {
```

I think the key is `session_storage`, here and in various places below,